gui/wxpython/gui_core/pyedit.py

```diff
@@ -592,6 +592,11 @@ def __init__(self, parent, giface, id=wx.ID_ANY,
             model=MenuTreeModelBuilder(filename).GetModel(separators=True))
         self.SetMenuBar(self.menubar)
 
+        menu = self.menubar.GetMenu(0)
```

Comment:
Please add comment on which menu it is expecting to find using index 0.

Reply:
I've already fixed it.